lib/showcase_generator.dart

```diff
@@ -62,10 +62,7 @@ class _ShowcaseGenerator extends Generator {
     }, orElse: () => null);
     final bool hasAnyRequiredParameter = _firstRequiredParameter != null;
 
-    final ConstructorElement forDesignTime = element.constructors.singleWhere(
-      (ConstructorElement c) => c.name == 'forDesignTime',
-      orElse: () => null,
-    );
+    final MethodElement forDesignTime = element.getMethod('forDesignTime');
```

Comment:
Does this work for factories? :cool:

Reply:
I didn't have lucky on using either `getMethod` or `getNamedConstructor` for factories :(